penguin/engine.py

```
@@ -1904,268 +1969,68 @@ async def run_task(
+            }
 
-        except Exception as e:
-            # Handle any execution errors
-            logger.error(f"Error executing task: {str(e)}")
-            completion_status = "error"
+            all_completion_phrases = [TASK_COMPLETION_PHRASE]
+            if completion_phrases:
+                all_completion_phrases.extend(completion_phrases)
+
+            async def task_message_callback_wrapper(
+                chunk: str, message_type: str, *args: Any, **kwargs: Any
```

Ruff (0.15.10)

[warning] Unused function argument: `args`

(ARG001)